docs/components/select.md

```diff
@@ -44,7 +44,7 @@ Prop for providing select item options. Supports grouping items under one group
 interface SelectItem {
   label: string
   value: string | number
-  key?: string
+  key?: string // optional parameter that will be appended with `-selected` when selected
```

Comment:
should you add the comment as a block comment on the actual interface as well?

Reply:
Done